sites/skeleton.dev/src/routes/blog/types.ts

```diff
@@ -0,0 +1,93 @@
+export type BlogList = {
```

Comment:
@Hugos68 this type is missing a `vercelEnv: any;` prop, we can just add `vercelEnv: unknown;` to avoid using `any` here. this solves the svelte-check error.